micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/ExecutorServiceMetrics.java

```diff
@@ -59,6 +59,8 @@ public class ExecutorServiceMetrics implements MeterBinder {
 
     private static final String DEFAULT_EXECUTOR_METRIC_PREFIX = "";
 
+    public static final String DESCRIPTION_POOL_SIZE = "The current number of threads in the pool";
```

Comment:
Let's keep it private unless there's a good reason for it to be public.

Reply:
Oh, sorry, something wrong with my autocomplete. Fixed